pkg/apis/deployment/v1alpha/deployment_spec.go

```diff
@@ -277,6 +290,9 @@ func (s *DeploymentSpec) Validate() error {
 	if err := s.License.Validate(); err != nil {
 		return maskAny(errors.Wrap(err, "spec.licenseKey"))
 	}
+	if err := k8sutil.ValidateResourceName(s.GetRootUserAccessSecretName()); err != nil {
```

Comment:
I expect that I can also set it to None.

Reply:
Explicitly setting this to `None` means that you do not want the operator to set any root password.
Therefore this should NOT be the default.